forge.config.ts

```diff
@@ -106,8 +106,6 @@ const config: ForgeConfig = {
 			// By default the dev server uses the same port as calypso.localhost
 			port: 3456,
 		} ),
-		// This plugin bundles the externals defined in the Webpack config file.
```

Comment:
Let's add comment about - "why do we need it, in our case?", instead of "What does it do?".
Since now we have just description of the plugin, what is useless for devs who will be working around this code in the future.

Reply:
Sorry I don't get it fully, this comment was removed, so where should we add this explanation?